app/src/main/java/com/example/happybirthday/unit3/ui/game/GameFragment.kt

```diff
@@ -0,0 +1,127 @@
+package com.example.happybirthday.unit3.ui.game
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import com.example.happybirthday.R
+import com.example.happybirthday.databinding.GameFragmentBinding
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+
+class GameFragment : Fragment() {
+
+    private val viewModel: GameViewModel by viewModels()
+
+
+    // Binding object instance with access to the views in the game_fragment.xml layout
```

Comment:
Such comments are usually useless. In most cases if someone else looks at code, they understand it. 
There are two places where comments are needed usually:
1. public code in libraries (SDK) - so integrators know what the code does and how to use it
2. some mathematically complicated code / workarounds - it's usually hard to understand complex math

Reply:
you don't have to remove them now, just don't add them in future